### PR TITLE
Add endpoint to get MTUs by update UUID.

### PR DESCRIPTION
**Pull request by houcros**

### CODE REVIEW

File: src/main/scala/com/advancedtelematic/director/db/SetMultiTargets.scala [hunk 1]
```diff
@@ -72,10 +72,18 @@ class SetMultiTargets()(implicit messageBusPublisher: MessageBusPublisher) exten
     dbAct.transactionally
   }
 
+  def getMultiTargetUpdates(namespace: Namespace, updateId: UpdateId)(implicit db: Database, ec: ExecutionContext): Future[MultiTargetUpdateRequest] = {
+    multiTargetUpdatesRepository
+      .fetch(updateId, namespace)
+      .map(_.map(mtuRow => mtuRow.hardwareId -> TargetUpdateRequest(mtuRow.fromTarget, mtuRow.toTarget, mtuRow.targetFormat, mtuRow.generateDiff)))
+      .map(_.toMap)
+      .map(MultiTargetUpdateRequest)
+  }
```
Comment (simao) [inline src/main/scala/com/advancedtelematic/director/db/SetMultiTargets.scala]: This class should still handle only *Row data types (see other methods, they use that) and the conversion to request should only be done on the http level, in this case on the resource

Comment (houcros) [reply src/main/scala/com/advancedtelematic/director/db/SetMultiTargets.scala]: Oh I see what you mean. I've updated now.

**Comment by houcros** [comment]
Rebased and squashed commits.

File: src/main/scala/com/advancedtelematic/director/http/AdminResource.scala [hunk 2]
```diff
@@ -257,6 +267,9 @@ class AdminResource(extractNamespace: Directive1[Namespace], val keyserverClient
 
   def multiTargetUpdatesRoute(ns: Namespace): Route =
     pathPrefix("multi_target_updates" / UpdateId.Path) { updateId =>
+      (pathEnd & get) {
```
Comment (jochenschneider) [inline src/main/scala/com/advancedtelematic/director/http/AdminResource.scala]: You can factor out `pathEnd`.

Comment (houcros) [reply src/main/scala/com/advancedtelematic/director/http/AdminResource.scala]: I did consider it, but in this case it seems more readable like this...

**Comment by jerrytrieu** [comment]
Let's merge.

**Pull request merged**